packages/server/utils/analytics/amplitude/AmplitudeAnalytics.ts

```diff
@@ -16,6 +16,21 @@ export class AmplitudeAnalytics {
     })
   }
 
+  identify(userId: string, traits: Record<string, any>, anonymousId?: string) {
```

Comment:
-1 Let's specify the traits to avoid developer mistakes when using this call. While we're at it, we can also include the `userId` so we just have one options object
```ts
type IdentifyOptions = {
  userId: string
  email: string
  name?: string
  isActive?: boolean
  featureFlags?: string[]
  highestTier?: string
  isPatient0?: boolean
  createdAt?: Date
}
```